README-SW.md

```diff
@@ -29,17 +29,17 @@
         <li><a href="./README-NE.md">नेपाली</a></li>
     </ul>
 </details>
-<!-- Do not translate this -->
+<!-- Tafadhali usitafsiri hii -->
 
 # Karibu Wachangiaji Wapya wa Open Source!
 
-[![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](https://makeapullrequest.com)
-[![first-timers-only Friendly](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](https://www.firsttimersonly.com/)
-[![Check Resources](https://github.com/freeCodeCamp/how-to-contribute-to-open-source/actions/workflows/test.yml/badge.svg)](https://github.com/freeCodeCamp/how-to-contribute-to-open-source/actions/workflows/test.yml)
+[![Pull Requests Karibu](https://img.shields.io/badge/PRs-karibu-brightgreen.svg?style=flat)](https://makeapullrequest.com)
```

Comment:
the links should remain the same without translating them,
translating links would lead to having broken links